android/src/main/java/app/notifee/core/NotifeeAlarmManager.java

```diff
@@ -140,9 +140,11 @@ static void scheduleTimestampTriggerNotification(
     AlarmManager alarmManager = getAlarmManager();
 
     // Verify we can call setExact APIs to avoid a crash, but it requires an Android S+ symbol
-    if ((Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) && !alarmManager.canScheduleExactAlarms()) {
-      System.err.println("Missing SCHEDULE_EXACT_ALARM permission. Trigger not scheduled. Issue #239");
-      return;
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
rewrote to this as android studio editor highlights this as an error even though it will still run successfully

Reply:
LGTM :-) - I did test it but that was a vscode edit vs the main dev of this crash fix which was in studio so I didn't see it